### PR TITLE
Missing 'resourceGroupName' parameter in example 1,2,7 and 8

### PR DESCRIPTION
**Pull request by isshukla**
Submitted via Docs Editor.

### CODE REVIEW

File: src/Synapse/Synapse/help/Set-AzSynapseSqlAuditSetting.md [hunk 1]
```diff
@@ -43,82 +47,94 @@ Set-AzSynapseSqlAuditSetting [-AuditActionGroup <AuditActionGroups[]>] [-Predica
...
 The **Set-AzSynapseSqlAuditSetting** cmdlet changes the auditing settings of an Azure Synapse Analytics Workspace.
 When blob storage is a destination for audit logs, specify the *StorageAccountResourceId* parameter to determine the storage account for the audit logs and the *StorageKeyType* parameter to define the storage keys. You can also define retention for the audit logs by setting the value of the *RetentionInDays* parameter to define the period for the audit logs.
 
 ## EXAMPLES
 
 ### Example 1
+
 ```powershell
-Set-AzSynapseSqlAuditSetting -WorkspaceName ContosoWorkspace -BlobStorageTargetState Enabled -StorageAccountResourceId "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/resourcegroup01/providers/Microsoft.Storage/storageAccounts/mystorage" -StorageKeyType Primary
+Set-AzSynapseSqlAuditSetting -WorkspaceName ContosoWorkspace -ResourceGroupName ContosoResourceGroup -BlobStorageTargetState Enabled -StorageAccountResourceId "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/resourcegroup01/providers/Microsoft.Storage/storageAccounts/mystorage" -StorageKeyType Primary
```
Comment (BethanyZhou) [inline src/Synapse/Synapse/help/Set-AzSynapseSqlAuditSetting.md:59]: From the syntax of `Set-AzSynapseSqlAuditSetting`, `ResourceGroupName` is an optional parameter, @wonner , could you help verify current change?

File: src/Synapse/Synapse/help/Set-AzSynapseSqlAuditSetting.md [hunk 2]
```diff
@@ -43,82 +47,94 @@ Set-AzSynapseSqlAuditSetting [-AuditActionGroup <AuditActionGroups[]>] [-Predica
...
+
 ```powershell
-Set-AzSynapseSqlAuditSetting -WorkspaceName ContosoWorkspace -LogAnalyticsTargetState Enabled -WorkspaceResourceId "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/insights-integration/providers/Microsoft.OperationalInsights/workspaces/myworkspace"
+Set-AzSynapseSqlAuditSetting -WorkspaceName ContosoWorkspace -ResourceGroupName ContosoResourceGroup -LogAnalyticsTargetState Enabled -WorkspaceResourceId "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/insights-integration/providers/Microsoft.OperationalInsights/workspaces/myworkspace"
 ```
 
 Enable the log analytics auditing policy of an Azure Synapse Analytics Workspace named ContosoWorkspace.
 
 ### Example 8
+
```
Comment (BethanyZhou) [inline src/Synapse/Synapse/help/Set-AzSynapseSqlAuditSetting.md:113]: No need for extra line in MDs, could you revert these changes?

File: src/Synapse/Synapse/help/Set-AzSynapseSqlAuditSetting.md (hunk 1 above)
Comment (wonner) [reply src/Synapse/Synapse/help/Set-AzSynapseSqlAuditSetting.md:59]: Hi @BethanyZhou @isshukla It's a bug need to fix. For now `ResourceGroupName` is a required parameter. @yanjungao718 please help fix, let's keep `ResourceGroupName` as optional.

Comment (yanjungao718) [reply src/Synapse/Synapse/help/Set-AzSynapseSqlAuditSetting.md:59]: ok, I'll make the change.

**Comment by BethanyZhou** [comment]
Close this PR as https://github.com/Azure/azure-powershell/pull/18983 is changing ResourceGroupName as optional. Please feel free to reopen this PR.
